shared/actions/chat/index.js

```diff
@@ -342,7 +343,7 @@ function * _loadMoreMessages (action: Constants.LoadMoreMessages): SagaGenerator
       },
       query: {
         disableResolveSupersedes: false,
-        markAsRead: true,
+        markAsRead: appFocused,
```

Comment:
I think this also needs the selected conversation state. We don't want to mark it if we call this programmatically on a non selected thread